R/ecoretriever.R

```diff
@@ -131,16 +131,53 @@ download = function(dataset, path='.', log_dir=NULL) {
 #' @return returns a character vector with the available datasets for download
 #' @export
 #' @examples 
+#' \donttest{
 #' ecoretriever::datasets()
+#' }
 datasets = function(){
   system('retriever ls', intern = TRUE) 
 }
 
+#' Update the retriever's dataset scripts to the most recent versions.
+#' 
+#' This function will check if the version of the retriever's scripts in your local
+#' directory \file{~/.retriever/scripts/} is up-to-date with the most recent official
+#' retriever release. Note it is possible that even more updated scripts exist
+#' at the retriever repository \url{https://github.com/weecology/retriever/tree/master/script}
+#' that have not yet been incorperated into an official release, and you should 
+#' consider checking that page if you have any concerns. 
+#' @keywords utilities
+#' @export
+#' @examples
+#' \donttest{
+#' ecoretriever::get_updates()
+#' }
+get_updates = function() {
+    cat('Please wait while the retriever updates its scripts, ...\n')
```

Comment:
I'd still recommend you use `writeLines(strwrap("string"))` in place of `cat("string")`. Some of your messages are long and you're assuming people running this will have a reasonably wide console. I know it's a little more verbose but you gain a lot of flexibility because of this.